add partial to warn people that code in plugin file runs in Node (#3686)
cypress-io/cypress-documentation: source/_partial/code_runs_in_node.md

{% note warning %}

> ⚠️ This code is part of the [plugin file](/guides/core-concepts/writing-and-organizing-tests.html#Plugin-files) and thus executes in Node environment. There are no `Cypress` or `cy` objects, but you do have the direct access to the file system and the rest of the operating system.

Reviewer: ```suggestion
⚠️ This code is part of the [plugin file](/guides/core-concepts/writing-and-organizing-tests.html#Plugin-files) and thus executes in the Node environment. You cannot call `Cypress` or `cy` commands in this file, but you do have the direct access to the file system and the rest of the operating system.
```
